.github/workflows/ci.yaml

```diff
@@ -2,6 +2,7 @@ on:
   push:
     branches:
       - master
+      - angular-skeleton
   pull_request:
 name: ci
```

Comment:
overwrite frontend-tests with your own tests:

```yaml
 frontend-test:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        node: [12]
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: ${{ matrix.node }}
      - run: node --version
        working-directory: ./packages/frontend
      - run: npm install
        working-directory: ./packages/frontend
      - run: npm test
        working-directory: ./packages/frontend
```

☝️   make sure we run unit, and component, and linting.